mlflow/pipelines/cards/pandas_renderer.py

```diff
@@ -200,23 +200,26 @@ def convert_to_comparison_proto(
 
 
 def construct_facets_html(
-    proto: facet_feature_statistics_pb2.DatasetFeatureStatisticsList, compare: bool = False
+    proto: facet_feature_statistics_pb2.DatasetFeatureStatisticsList,
+    compare: bool = False,
+    chart_selection: str = "Standard",
 ) -> str:
     """
     Constructs the facets HTML to visualize the serialized FeatureStatisticsList proto.
     :param proto: A DatasetFeatureStatisticsList proto which contains the statistics for a DataFrame
     :param compare: If True, then the returned visualization switches on the comparison
         mode for several stats.
+    :param chart_selection: The selected histogram chart by default
     :return: the HTML for Facets visualization
     """
     # facets_html_bundle = _get_facets_html_bundle()
     protostr = base64.b64encode(proto.SerializeToString()).decode("utf-8")
     html_template = """
         <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.3.3/webcomponents-lite.js"></script>
-        <link rel="import" href="https://raw.githubusercontent.com/PAIR-code/facets/1.0.0/facets-dist/facets-jupyter.html" >
```

Comment:
Did you fork this HTML? It could make future facet upgrade difficult. Any options to keep the upstream? WDYT? @sunishsheth2009

Reply:
I couldn't find a solution for doing this without forking this. 
I am curious how important this is which could cause upgrading issues in the future? 
If we want to do this, can we copy paste the code in mlflow folder itself or fork it under mlflow which would help with better upgrade support I think.

Reply:
+1 - the forked code should live in MLflow

Reply:
This code was copied to MLflow.
The URL is for getting raw file from github, similar to the existing one, which is a raw file from github.

There will be another PR to replace mingyu89/mlflow with mlflow/mlflow in the path after this one is merged.